tests/php/Services/LabelSourceAdapters/WormsAdapterTest.php

```diff
@@ -119,6 +121,28 @@ public function testFindEmpty()
         $this->assertEquals([], $results);
     }
 
+    public function testFindThrowException()
+    {
+
+        $mock = Mockery::mock(SoapClient::class);
+        $mock->shouldReceive('getAphiaRecords')
+            ->once()
+            ->andThrow(new ServiceUnavailableHttpException(Response::HTTP_SERVICE_UNAVAILABLE, 'test'));
+
+        $adapter = new WormsAdapter;
+        $adapter->setSoapClient($mock);
+
+        $request = new Request;
+        $request->merge(['query' => 'Kolga']);
+
+        try {
+            $adapter->find($request);
+        } catch(ServiceUnavailableHttpException $e) {
+            $this->assertEquals(Response::HTTP_SERVICE_UNAVAILABLE, $e->getStatusCode());
+            $this->assertEquals('test', $e->getMessage());
+        }
```

Comment:
The test would pass if no error was thrown, too. In cases like these, I do:

```suggestion
        try {
            $adapter->find($request);
            $this->assertFalse(true);
        } catch(ServiceUnavailableHttpException $e) {
            $this->assertEquals('The WoRMS server is currently unavailable.', $e->getMessage());
        }
```